public/configs/v1/env.json

```diff
@@ -491,6 +492,11 @@
                "minimalVersion": "1.0",
                "build": 40000,
                "url": "https://apps.apple.com/app/dydx/id1564787350"
+            },
+            "android": {
+               "minimalVersion": "1.1",
```

Comment:
should we just make this a very high number so that it's always the case? Can u also change for iOS as well.